### ISSUE TITLE
Symfony 6.3

### ISSUE THREAD

**Issue by pfpro** [craue/CraueConfigBundle#61]
Hi there,

I try to upgrade from Symfony 6.2 to 6.3 and get the following logs:

```
Method "Symfony\Component\DependencyInjection\Extension\ExtensionInterface::load()" might add "void" as a native return type declaration in the future. Do the same in implementation "Craue\ConfigBundle\DependencyInjection\CraueConfigExtension" now to avoid errors or add an explicit @return annotation to suppress this message.

Method "Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface::prepend()" might add "void" as a native return type declaration in the future. Do the same in implementation "Craue\ConfigBundle\DependencyInjection\CraueConfigExtension" now to avoid errors or add an explicit @return annotation to suppress this message.

Method "Symfony\Component\Form\AbstractType::buildForm()" might add "void" as a native return type declaration in the future. Do the same in child class "Craue\ConfigBundle\Form\Type\SettingType" now to avoid errors or add an explicit @return annotation to suppress this message.

Method "Symfony\Component\Form\AbstractType::finishView()" might add "void" as a native return type declaration in the future. Do the same in child class "Craue\ConfigBundle\Form\Type\SettingType" now to avoid errors or add an explicit @return annotation to suppress this message.

Method "Symfony\Component\Form\AbstractType::configureOptions()" might add "void" as a native return type declaration in the future. Do the same in child class "Craue\ConfigBundle\Form\Type\SettingType" now to avoid errors or add an explicit @return annotation to suppress this message.
```

Any chance you can fix this?

Best regards

**Comment by craue** [comment]
I've just released version 2.7.0 which should run silently with Symfony 6.3.

**Comment by craue** [comment]
fixed in 11eb274b1aeee12029aba186ae0723087051de37
